director/lib/conversion_types.py

```diff
@@ -18,7 +18,17 @@ class UnknownMimeTypeError(ValueError):
 class ConversionFormat(typing.NamedTuple):
     format_id: str
     mimetypes: typing.Iterable[str]
-    output_extension: typing.Optional[str] = None
+    extensions: typing.List[str] = []
+
+    @property
+    def default_extension(self) -> str:
+        """Get the default extension for the format."""
+        return self.extensions[0] if len(self.extensions) else self.format_id
```

Comment:
It's more Pythonic just to do `if self.extensions` which will be `False` if the list is empty.